### DISCUSSION TITLE
Homepage link on pub.dev

### DISCUSSION THREAD

**Discussion by agata** [singerdmx/flutter-quill#2207] [General]
The link to the Homepage in the sidebar of https://pub.dev/packages/flutter_quill is directed to https://1o24bbs.com/c/bulletjournal/108/, but it seems that this page is no longer active.

When I first discovered this library, I followed the Homepage link and got the impression that the library was no longer maintained. To prevent others from hesitating to use the library due to the Homepage, I would suggest removing the link to the Homepage on pub.dev, if possible.

**Comment by EchoEllet** [comment]
Thank you for the feedback.

![image](https://github.com/user-attachments/assets/253fbc45-9908-4ab6-8107-70e689b04410)

You can find how long ago it was last updated on the pub.dev and usually, it will be the first thing the user sees.

Since this link is no longer available, will replace it with the GitHub repository link for now.

**Comment by agata** [comment]
Thank you for your prompt response.

As you mentioned, I also checked the last updated date and confirmed that the library is active before deciding to adopt it. I appreciate your quick action.
